examples/video_channel_create.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import logging
+import sys
+
+import jwplatform
+
+
+def create_channel(api_key, api_secret, channel_type='manual', **kwargs):
+    """
+    Function which creates a new channel. Channels serve as containers of video/media objects.
+
+    :param api_key: <string> JWPlatform api-key
+    :param api_secret: <string> JWPlatform shared-secret
+    :param channel_type: <string> REQUIRED Acceptable values include 'manual','dynamic','trending','feed','search'
+    :param kwargs: Arguments conforming to standards found @ https://developer.jwplayer.com/jw-platform/reference/v1
```

Comment:
Could we deep link these to actual endpoint?  

for example, this one would link to https://developer.jwplayer.com/jw-platform/reference/v1/methods/channels/create.html